LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
Why are we removing the MIT license? This code is open source and free to use/ change.